codebases/openzaak.html

```diff
@@ -0,0 +1,116 @@
+---
+layout: codebase
+title: Codebases we work with - OpenZaak
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s2">
+      <div class="cb-logo"><img src="../photos/codebases/openzaak/openzaak_logo_small.png" alt="OpenZaak logo"></div>
+      <h1>OpenZaak</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s2">Stage 2: <strong>Incubation</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>OpenZaak is a case management platform for municipalities that want to deliver citizen-friendly services online.</p>
+      <p>It is a ready-to-use backend solution that acts as a directory of case registration, including metadata describing the case.</p>
+      <p>OpenZaak implements the <a href="https://www.vngrealisatie.nl/producten/api-standaarden-zaakgericht-werken" target="_blank">Dutch national API standards for case management</a>, developed by the Dutch Association of Municipalities (VNG).</p>
+    </div>
+
+    <div class="cb-detail-gallery">
+      <a href="#img1">
+          <img class="lb-thumb" src="../photos/codebases/openzaak/openzaak_thumb_g01.png" alt="OpenZaak product thumbnail">
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <!--<a href="#img3" class="lb-btn lb-btn-prev"></a>-->
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img1"><img src="../photos/codebases/openzaak/openzaak_g01.png" alt="OpenZaak product example 1"></a>
```

Comment:
```suggestion
        <a href="#img1"><img src="../photos/codebases/openzaak/openzaak_g01.png" alt="Configuration menu of OpenZaak displaying a date picker"></a>
```

Reply:
Thank you for writing these much improved alt tags @Ainali !!